spec/datadog/core/crashtracking/component_spec.rb

```diff
@@ -138,12 +138,24 @@
         end
       end
 
-      it 'update_on_fork the crash tracker' do
+      it 'updates the crash tracker' do
         expect(described_class).to receive(:_native_start_or_update_on_fork).with(
           hash_including(action: :update_on_fork)
         )
 
-        crashtracker = build_crashtracker
+        crashtracker = build_crashtracker(logger: logger)
+
+        crashtracker.update_on_fork
+      end
+
+      it 'refreshes the latest settings' do
+        expect(Datadog).to receive(:configuration).and_return(:latest_settings)
+        expect(Datadog::Core::Crashtracking::TagBuilder).to receive(:call).with(:latest_settings).and_return([:latest_tags])
```

Comment:
I think it's very minor thing, but technically this is not an expectation, but rather allowance. Not critical, but I would split expectations from the setup, like this

```suggestion
        allow(Datadog).to receive(:configuration).and_return(:latest_settings)
        allow(Datadog::Core::Crashtracking::TagBuilder).to receive(:call).with(:latest_settings).and_return([:latest_tags])

```

Reply:
Yeah -- I'll admit I'm sometimes slightly inconsistent about remembering to do this, although I agree with your reasoning. Applied in 06dfc27b6001ea1d621c4e87331de29297ff087f